react-basic-practice/src/components/TableFinance/index.tsx

```diff
@@ -16,9 +16,11 @@ const TableFinance: React.FC<ITablePrice> = ({
 
   return (
     <Box fontWeight="normal">
-      <Text fontSize="md" lineHeight="short" color={color}>
-        {value}
-      </Text>
+      <Box display="inline-flex" fontSize="md" lineHeight="short" color={color}>
+        <Text>$</Text>
```

Comment:
Any styles on this, please check the same

Reply:
i done